doc/modules/ROOT/pages/model-object.adoc

```diff
@@ -104,15 +144,15 @@ Once trained, in addition to the xref:model-object.adoc#model-object-inspecting[
 | metrics        | -                | Series      | Returns values for the metrics computed when training.
 |===
 
-Suppose then that we have a trained GraphSAGE model `gs_model` and a graph `H` for which we would like to derive node embeddings. Then we could do the following:
+So given the GraphSAGE model `model` we trained above, we could do the following:
 
-[source,python]
+[source,python,group=gs-model]
 ----
 # Make sure our training actually converged
-metrics = gs_model.metrics()
+metrics = model.metrics()
 assert metrics["didConverge"]
```

Comment:
I found the bug finally!! I don't understand how this ran green for you, because there is a missing step here:

```
_ = gds.alpha.model.load(model.name())
```

This is necessary because in the previous example we stored and dropped the model. It must be loaded for the `predict_write` to work.

Reply:
I can also see it locally now. Probably switched to Adam sides before enabling this part :see_no_evil: 

Thank you for finding it!